pkg/security/probe/discarders.go

```diff
@@ -170,6 +188,18 @@ func newInodeDiscarders(inodesMap, revisionsMap *lib.Map, erpc *ERPC, dentryReso
 	return id, nil
 }
 
+func (id *inodeDiscarders) isRecentlyAdded(mountID uint32, inode uint64, timestamp uint64) bool {
+	entry := id.recentlyAddedEntries[recentlyAddedIndex(mountID, inode)]
+	return entry.MountID == mountID && entry.Inode == inode && timestamp-entry.Timestamp < recentlyAddedTimeout
```

Comment:
This is a nit but I think `timestamp-entry.Timestamp` can overflow if we have reordering issues on the perf map.